crates/belly_core/src/input.rs

```diff
@@ -15,47 +14,30 @@ impl Plugin for ElementsInputPlugin {
         app.add_event::<PointerInput>()
             .add_event::<RequestFocus>()
             .init_resource::<Focused>()
-            .add_system(
-                pointer_input_system
-                    .in_base_set(CoreSet::PreUpdate)
-                    .in_set(Label::Signals)
-                    .after(InputSystem),
+            .add_systems(
```

Comment:
Can you please add comments on what is happening here? This is the most complex place in whole PR and seems like it's a very important, it defines input processing pipeline, I can't comprehend.